docs/protocol/architecture.md

```diff
@@ -24,17 +22,17 @@ The following process describes, step by step, how a contract is submitted and a
 
-This encrypted data can only be accessed from within the “trusted execution environment”, or enclave, that the `compute` module requires each validator to run. The computation indicated by the Secret Contract is then performed, within this trusted enclave, over the decrypted data. When the computation is completed, the output is encrypted and recorded on-chain. There are various types of outputs that can be expected. These include:
+This encrypted data can only be accessed from within the “trusted execution environment”, or enclave, that the `wasm` module requires each validator to run. The computation indicated by the Secret Contract is then performed, within this trusted enclave, over the decrypted data. When the computation is completed, the output is encrypted and recorded on-chain. There are various types of outputs that can be expected. These include:
 
 - An updated contract state (i.e., the user’s data should update the state or be stored for future computations)
 - A computation result encrypted for the transaction sender (i.e., a result should be returned privately to the sender)
 - Callbacks to other contracts (i.e., a contract is called conditional on the outcome of a Secret Contract function)
-- Send Messages to other modules (i.e., for sending value transfer messages that depend on the outcome of a computation). Currently a contract can only queue "send funds" msg and "invoke another contract" msg. See [from cosmwasm code](https://github.com/enigmampc/SecretNetwork/blob/e1c25ed06a9b3abba0378bdd858bad376dd828c9/cosmwasm/src/types.rs#L99-L112)
+- Send Messages to other modules (i.e., for sending value transfer messages that depend on the outcome of a computation). See [from go-cosmwasm code](https://github.com/enigmampc/SecretNetwork/blob/master/go-cosmwasm/types/msg.go#L63-L69)
 
 The Secret Network’s `compute` module currently requires that validators run nodes with Intel SGX chips (enclaves). These enclaves contain signing keys that are generated within the enclave. For more details on how enclaves function and are verified, see [intel SGX](sgx.md).
 
 ![enclave](../images/diagrams/enclave.png)
 
-Diagram: Trusted and Untrusted aspects of Secret Network code. `compute` enables cosmwasm with encryption to be executed within the trusted component of the enclave.
+Diagram: Trusted and Untrusted aspects of Secret Network code. `wasm` enables go-cosmwasm with encryption to be executed within the trusted component of the enclave.
```

Comment:
Again, compute

Reply:
Good to know, thanks Assaf! I wasn't sure which one was canon.